crates/goose/src/providers/anthropic.rs

```diff
@@ -71,7 +74,13 @@ impl AnthropicProvider {
         })
     }
 
-    pub fn from_custom_config(model: ModelConfig, config: CustomProviderConfig) -> Result<Self> {
+    pub fn from_custom_config(
+        mut model: ModelConfig,
+        config: CustomProviderConfig,
+    ) -> Result<Self> {
+        // Set the default fast model for Anthropic
+        model.fast_model = Some("claude-3-5-haiku-latest".to_string());
```

Comment:
you don't want to do this - custom_config means it is a provider that speaks anthropic but isn't. they will not have this model